libindy/src/api/payments.rs

```diff
@@ -454,34 +455,31 @@ pub extern fn indy_list_payment_addresses(command_handle: i32,
 }
 
 /// Modifies Indy request by adding information how to pay fees for this transaction
-/// according to selected payment method.
-///
-/// Payment selection is performed by looking to o
+/// according to this payment method.
 ///
-/// This method consumes set of UTXO inputs and outputs. The difference between inputs balance
+/// This method consumes set of inputs and outputs. The difference between inputs balance
 /// and outputs balance is the fee for this transaction.
 ///
 /// Not that this method also produces correct fee signatures.
 ///
 /// Format of inputs is specific for payment method. Usually it should reference payment transaction
 /// with at least one output that corresponds to payment address that user owns.
 ///
+///
 /// Note this endpoint is EXPERIMENTAL. Function signature and behaviour may change
 /// in the future releases.
 ///
 /// #Params
 /// wallet_handle: wallet handle
 /// submitter_did : DID of request sender
 /// req_json: initial transaction request as json
-/// inputs_json: The list of UTXO inputs as json array:
-///   ["input1", ...]
-///   Notes:
-///     - each input should reference paymentAddress
-///     - this param will be used to determine payment_method
```

Comment:
@Artemkaaas @vimmerru are we actually would like to remove the note about way to determine payment method?

Reply:
I suggest to keep it